the-blue-alliance-ios/App Delegate/AppDelegate.swift

```diff
@@ -23,72 +24,42 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
-        let districtsViewController = DistrictsContainerViewController(myTBA: myTBA,
-                                                                       statusService: statusService,
-                                                                       urlOpener: urlOpener,
-                                                                       persistentContainer: persistentContainer,
-                                                                       tbaKit: tbaKit,
-                                                                       userDefaults: userDefaults)
-        let settingsViewController = SettingsViewController(fcmTokenProvider: messaging,
-                                                            myTBA: myTBA,
-                                                            pushService: pushService,
-                                                            searchService: searchService,
-                                                            urlOpener: urlOpener,
-                                                            persistentContainer: persistentContainer,
-                                                            tbaKit: tbaKit,
-                                                            userDefaults: userDefaults)
-        let myTBAViewController = MyTBAViewController(myTBA: myTBA,
-                                                      statusService: statusService,
-                                                      urlOpener: urlOpener,
-                                                      persistentContainer: persistentContainer,
-                                                      tbaKit: tbaKit,
-                                                      userDefaults: userDefaults)
-        let rootViewControllers: [UIViewController] = [eventsViewController, teamsViewController, districtsViewController, myTBAViewController, settingsViewController]
-        tabBarController.viewControllers = rootViewControllers.compactMap({ (viewController) -> UIViewController? in
-            let navigationController = UINavigationController(rootViewController: viewController)
-            return navigationController
-        })
-
-        splitViewController.viewControllers = [tabBarController, emptyNavigationController]
-
-        splitViewController.preferredDisplayMode = .allVisible
-        splitViewController.delegate = self
-
-        return splitViewController
-    }()
-    private let tabBarController = UITabBarController()
-    lazy var emptyNavigationController: UINavigationController = {
-        guard let emptyViewController = Bundle.main.loadNibNamed("EmptyViewController", owner: nil, options: nil)?.first as? UIViewController else {
-            fatalError("Unable to load empty view controller")
+    // Root VC on iPhone is a tab bar controller
+    lazy private var rootViewController: UIViewController & RootController = {
+        switch UIDevice.current.userInterfaceIdiom {
```

Comment:
We might want to move this code in to some `UIDevice+TBA` extension in order to have some `isPad` and `isPhone` booleans we can use moving forward when we're working on different layouts for the detail views on iPad